sdk/nodejs/cmd/pulumi-language-nodejs/main_test.go

```diff
@@ -165,6 +165,14 @@ func TestGetRequiredPlugins(t *testing.T) {
 			filepath.Join(dir, "node_modules", "malformed", "tests", "malformed_test", "package.json"),
 			`{`,
 		},
+		{
+			filepath.Join(dir, "node_modules", "malformed", "tests", "false_main", "package.json"),
+			`{ "name": "false_main", "main": false }`,
+		},
+		{
+			filepath.Join(dir, "node_modules", "malformed", "tests", "invalid_main", "package.json"),
+			`{ "name": "invalid_main", "main": ["why is this a thing", "srsly omg wtf"]}`,
```

Comment:
As funny as this line is, maybe a string saying what you expect to fail/not fail etc would help better haha